internal/http/activity.go

```diff
@@ -1,33 +1,31 @@
-package routes
+package http
 
 import (
 	"context"
 	"encoding/json"
+	"fmt"
 	"net/http"
 	"sublinks/sublinks-federation/internal/activitypub"
 	"sublinks/sublinks-federation/internal/lemmy"
-	"sublinks/sublinks-federation/internal/log"
-
-	"fmt"
 
 	"golang.org/x/text/cases"
 	"golang.org/x/text/language"
 
 	"github.com/gorilla/mux"
 )
 
-func SetupActivityRoutes(r *mux.Router) {
-	r.HandleFunc("/activities/{action}/{id}", getActivityHandler).Methods("GET")
+func (s Server) SetupActivityRoutes() {
+	s.Router.HandleFunc("/activities/{action}/{id}", s.getActivityHandler).Methods("GET")
 }
 
-func getActivityHandler(w http.ResponseWriter, r *http.Request) {
+func (s Server) getActivityHandler(w http.ResponseWriter, r *http.Request) {
```

Comment:
Maybe we can compromise and use `srv` for these? I really don't like non-descriptive variable names unless they are used in a for loop as an iterator (eg `for i = 0; i < 5; i++` type stuff) or in methods with only a couple of lines (so it's easy enough to keep track of what the variable means. Even then, it would probably be better if we standardized and used better naming everywhere. I know I'm guilty of this with the request/response writer variables)